src/Makefile.linux.mk

```diff
@@ -77,15 +77,14 @@ build/$1-%/lib/pkgconfig/capstone.pc: build/$1-env-%.rc build/.capstone-submodul
 		&& export PACKAGE_TARNAME=capstone \
 		&& . $$$$CONFIG_SITE \
 		&& case $$* in \
-			*-x86)    capstone_archs="x86"         ;; \
-			*-x86_64) capstone_archs="x86"         ;; \
-			*-arm)    capstone_archs="arm"         ;; \
-			*-armhf)  capstone_archs="arm"         ;; \
-			*-armeabi)capstone_archs="arm"         ;; \
-			*-arm64)  capstone_archs="aarch64"     ;; \
-			*-mips)   capstone_archs="mips"        ;; \
-			*-mips64) capstone_archs="mips64"      ;; \
```

Comment:
In this file we just add lots of MIPS64 entries which are similar to those for MIPS.